Lexplorer/Pages/NFTDetail.razor

```diff
@@ -132,6 +132,28 @@
         <MudTd DataLabel="Timestamp">@context.verifiedAt</MudTd>
     </RowTemplate>
 </MudTable>
+<br />
+<MudTable Dense="true" Striped="true" Bordered="true" Items="@holders" Hover="true" Loading=@isLoadingHolders>
+    <ToolBarContent>
+        <MudText Typo="Typo.h6">NFT Holders</MudText>
+        <MudSpacer />
+        <MudIconButton Icon="@Icons.Filled.FirstPage" Disabled=@(goToHolderPage == 1) OnClick="@(() => goToHolderPage = 1)" />
+        <MudIconButton Icon="@Icons.Filled.NavigateBefore" Disabled=@(goToHolderPage == 1) OnClick="@(() => goToHolderPage -= 1)" />
+        <MudNumericField HideSpinButtons="true" @bind-Value="goToHolderPage" Label="Page" Variant="Variant.Outlined" Min="1" Margin="Margin.Dense" Class="flex-grow-0" Style="width:50px;" />
+        <MudIconButton Icon="@Icons.Filled.NavigateNext" Disabled=@(holders!.Count < pageSize) OnClick="@(() => goToHolderPage += 1)" />
+        <MudSpacer />
+    </ToolBarContent>
+    <HeaderContent>
+        <MudTh>Account Id</MudTh>
+        <MudTh>Address</MudTh>
+        <MudTh>Balance</MudTh>
+    </HeaderContent>
+    <RowTemplate>
+        <MudTd DataLabel="Account Id">@context.Account.Id</MudTd>
```

Comment:
Needs ? operators, see warnings.

Reply:
Fixed

Reply:
No that didn't fix them. Instead of `@context.Account.id?` it should be `@context?.Account?.id`.

You should really check why you don't see those warnings!